coordinator/src/node/rollover.rs

```diff
@@ -215,6 +252,44 @@ impl Node {
             contract.get_temporary_id(),
         )
     }
+
+    pub fn rollback_channel_if_needed(
+        &self,
+        connection: &mut PgConnection,
+        signed_channel: SignedChannel,
+    ) -> Result<(bool, Option<ContractId>)> {
+        if is_channel_in_intermediate_state(&signed_channel) {
+            let trader_id = signed_channel.counter_party;
+            tracing::warn!(%trader_id, "Found signed channel contract in an intermediate state. Trying to rollback channel to the last stable state!");
```

Comment:
We could log the channel state as well - could be useful for debugging purposes to see in what stage users get stuck.